### DISCUSSION TITLE
How to reset all on pip installation

### DISCUSSION THREAD

**Discussion by Intranox** [open-webui/open-webui#5301] [Q&A] [answered]
I can't find the folder where the data is saved using pip installation. How can I reset everything? I don't remember the email I entered or the password

**Comment by m8dhouse** [comment, marked as answer]
you should do a search on you machine to find the open-webui folder (or the static subfolder)

something like this:
sudo find / -type d -name "open-webui" 2>/dev/null

or 

sudo find / -type d -name "static" 2>/dev/null

the 'main' folder is just below the static folder. It will be similar to this:
/usr/open-webui/venv/lib64/python3.11/site-packages/open_webui

venv would be the name of your virtual environment.

The data folder is here: /venv/lib64/python3.11/site-packages/data/
In here is the sqlite: webui.db

hope this helps

**Reply by Intranox** [reply]
Thanks, I found the file in .local/lib/python3.11/site-packages/data/webui.db and deleted it
